skin/frontend/rwd/default/css/styles.css

```diff
@@ -1072,6 +1072,20 @@ a.button:hover {
     float: none;
   }
 }
+
+/* -------------------------------------------- *
```

Comment:
we should update also the scssf files or this change will be lost when css will be regenerated.